packages/lesswrong/lib/components.js

```diff
@@ -100,12 +100,12 @@ import '../components/posts/MoveToDraft.jsx';
 import '../components/posts/SuggestAlignment.jsx';
 import '../components/posts/PostsItemMeta.jsx';
 import '../components/posts/PostsItem.jsx';
+import '../components/posts/PostsItemMetaInfo.jsx';
```

Comment:
Is there a reason for changing the order here? (This looks like it should have no effect; if it *does* have an effect, that's concerning.)